app/assets/stylesheets/components/_user_info_item.sass

```diff
@@ -0,0 +1,10 @@
+$UserInfoItem-color-delete: Theme-color(button, delete)
```

Comment:
This comment is about this whole file, not this specific line.
Please be careful with forgotten files. I believe this is not being used anymore.
Please check other possible cases like this